umap/settings/base.py

```diff
@@ -39,30 +39,34 @@
     ('ca', 'Catalan'),
     ('cs-cz', 'Czech'),
     ('da', 'Danish'),
-    ('de', 'Deutsch'),
+    ('de', 'German'),
     ('el', 'Greek'),
     ('en', 'English'),
-    ('es', 'Español'),
+    ('es', 'Spanish'),
     ('fi', 'Finnish'),
-    ('fr', 'Francais'),
+    ('fr', 'French'),
     ('gl', 'Galician'),
+    ('he', 'Hebrew'),
     ('hr', 'Croatian'),
     ('hu', 'Hungarian'),
-    ('it', 'Italiano'),
+    ('it', 'Italian'),
     ('ja', 'Japanese'),
     ('ko', 'Korean'),
     ('lt', 'Lithuanian'),
     ('nl', 'Dutch'),
     ('pl', 'Polish'),
     ('pt', 'Portuguese'),
+    ('pt-br', 'Portuguese (Brazil)'),
```

Comment:
Should we really have `pt`, `pt-br` and `pt-pt` ? Isn't a way to disperse translators energy ?
I mean, uMap is not literature, are Brazilian and portuguese from Portugal a that different for a technical site as this one ?
That may be a question to ask to the said translators ;)

Reply:
Since all PT translations are almost done, I think we can have those variants:
![image](https://user-images.githubusercontent.com/6597721/75422076-0da44280-593c-11ea-8ecd-7f776933464d.png)

Hi @alemenezesBR, @xendez, @joaoponceleao, since you are part of the [uMap Transifex team](https://www.transifex.com/openstreetmap/umap/languages/) and Portuguese speakers, can you please advise us here:

- Should we have pt, pt-br and pt-pt in uMap?
- Can you please have a look to [transifex](https://www.transifex.com/openstreetmap/umap/languages/) and tell us if strings are really differents for uMap in pt, pt-br and pt-pt.
- Any other comments/suggestions are very welcome 👍 

Have a nice day.

Reply:
I don't think there should be pt, pt-br and pt-pt is usually the way to go, and the reason these are usually together is that the same translator is translating both because the changes are pretty minor in writing as you mentioned but I would like to see what the translators think about that issue.

Reply:
We should only have pt-br and pt-pt. The main difference between portuguese from brazil and portugal is one of etiquette (i.e. sentence structure). Portugal is more formal, so a Brazilian phrase can be interpreted as a bit offensive. To make matters worse, what is considered formal in one idiom can be considered rude in the other, so its hard to have a single language to satisfy both. English does not have this difference (no 'tu' or 'vous'), so its mostly spelling issues there.

The only times where differences will show up tend to be in sentences asking for user input. Everywhere else the translations should be identical.

Reply:
Thanks a lot @joaoponceleao, we will keep only pt-br and pt-pt in uMap as you advised 👍
I think, I won't remove "pt" from Transifex, it can still be helpful.

Reply:
> We should only have pt-br and pt-pt. The main difference between portuguese from brazil and portugal is one of etiquette (i.e. sentence structure). Portugal is more formal, so a Brazilian phrase can be interpreted as a bit offensive. To make matters worse, what is considered formal in one idiom can be considered rude in the other, so its hard to have a single language to satisfy both. English does not have this difference (no 'tu' or 'vous'), so its mostly spelling issues there.
> 
> The only times where differences will show up tend to be in sentences asking for user input. Everywhere else the translations should be identical.

pt-br and pt-pt have big differences.
Certainly the language spoken in the two countries has many peculiarities, and even when comparing some regions, they may sound like two completely different languages.

It's a big mistake to say "Everywhere else the translations should be identical."